nbodykit/algorithms/convpower/catalog.py

```diff
@@ -200,7 +207,7 @@ def to_mesh(self, Nmesh=None, BoxSize=None, BoxCenter=None, dtype='f4', interlac
             deprecated. set nbar in the call to FKPCatalog()
         """
         from .catalogmesh import FKPCatalogMesh
-
+        
```

Comment:
Let's avoid introducing new extra spaces.

Reply:
Fixed this in latest commit.